src/SayMore/UI/ExportMediaDlg.cs

```diff
@@ -0,0 +1,260 @@
+using System;
+using System.ComponentModel;
+using System.Drawing;
+using System.Windows.Forms;
+using L10NSharp;
+using L10NSharp.UI;
+using SIL.Reporting;
+using SIL.Windows.Forms;
+using SIL.Windows.Forms.PortableSettingsProvider;
+using SayMore.Media.FFmpeg;
+using SayMore.Properties;
+
+namespace SayMore.UI
+{
+	public partial class ExportMediaDlg : Form
+	{
+		private readonly ConvertMediaDlgViewModel _viewModel;
+		private string _conversionInProgressStatusFormat;
+
+		public static DialogResult Show(string inputFile, string initialConversionName, string outputFile)
+		{
+			var viewModel = new ConvertMediaDlgViewModel(inputFile, initialConversionName, outputFile);
+			using (var dlg = new ExportMediaDlg(viewModel))
+				return dlg.ShowDialog();
+		}
+
+		/// ------------------------------------------------------------------------------------
+		public ExportMediaDlg()
+		{
+			InitializeComponent();
+			DialogResult = DialogResult.Cancel;
+		}
+
+		/// ------------------------------------------------------------------------------------
+		public ExportMediaDlg(ConvertMediaDlgViewModel viewModel) : this()
+		{
+			Logger.WriteEvent("ExportMediaDlg constructor. file = {0}", viewModel.InputFile);
+			_viewModel = viewModel;
+
+			if (Settings.Default.ExportMediaDlg == null)
+			{
+				StartPosition = FormStartPosition.CenterScreen;
+				Settings.Default.ExportMediaDlg = FormSettings.Create(this);
+			}
+
+			InitializeFonts();
+
+			if (_viewModel.MediaInfo == null)
+			{
+				_labelStatus.Text = LocalizationManager.GetString("DialogBoxes.ConvertMediaDlg.InvalidMediaFile",
```

Comment:
Any localizable strings in this file that are exact copies of the original ones in ConvertMediaDlg need to be turned into statics there and accessed wherever needed to avoid duplicates. Otherwise, if there is a reasonable possibility that text of the strings could diverge (either in English or in a localized version), they need to be given unique IDs in both places.

Reply:
Combined into one.